vary.js

```diff
@@ -1,16 +1,80 @@
 'use strict'
 
-const append = require('vary').append
+const LRUCache = require('mnemonist').LRUCache
+const cache = new LRUCache(1000)
+
+function parse (header) {
+  if (Array.isArray(header)) {
```

Comment:
I wonder if this is actually error prone and maybe should be removed if header can only be a string. We need to actually lowercase the values of the the header. Maybe converting the array with a join to a string and reparse it makes more sense.